ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXPerceptionVisualizersPanel.java

```diff
@@ -0,0 +1,122 @@
+package us.ihmc.rdx.ui.graphics;
+
+import com.badlogic.gdx.graphics.g3d.Renderable;
+import com.badlogic.gdx.utils.Array;
+import com.badlogic.gdx.utils.Pool;
+import imgui.ImGui;
+import imgui.type.ImString;
+import us.ihmc.rdx.imgui.ImGuiTools;
+import us.ihmc.rdx.imgui.RDXPanel;
+import us.ihmc.rdx.sceneManager.RDXRenderableProvider;
+import us.ihmc.rdx.sceneManager.RDXSceneLevel;
+
+import java.util.Comparator;
+import java.util.Locale;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class RDXPerceptionVisualizersPanel extends RDXPanel implements RDXRenderableProvider
+{
+   private static final String WINDOW_NAME = "Perception Visualizers";
+
+   private final TreeSet<RDXVisualizer> visualizers = new TreeSet<>(Comparator.comparing(RDXVisualizer::getTitle));
+   // Search filter
+   private final ImString filter = new ImString();
+   private boolean created = false;
+
+   public RDXPerceptionVisualizersPanel()
+   {
+      super(WINDOW_NAME);
+      setRenderMethod(this::renderImGuiWidgets);
+   }
+
+   public void addVisualizer(RDXVisualizer visualizer)
+   {
+      visualizers.add(visualizer);
+      RDXPanel panel = visualizer.getPanel();
+      if (panel != null)
+         addChild(panel);
+      if (created)
+         visualizer.create();
+   }
+
+   public void create()
+   {
+      for (RDXVisualizer visualizer : visualizers)
+      {
+         visualizer.create();
+      }
+      created = true;
+   }
+
+   public void update()
+   {
+      for (RDXVisualizer visualizer : visualizers)
+      {
+         if (visualizer.isActive())
+            visualizer.update();
+      }
+   }
+
+   public void renderImGuiWidgets()
+   {
+      if (ImGui.inputText("Search##perceptionVisualizers", filter))
+      {
+      }
+      ImGui.sameLine();
+      boolean collapse = false;
+      if (ImGui.button("Collapse##perceptionVisualizers"))
```

Comment:
I liike IntelliJ's suggestion to change this to 
```
boolean collapse = ImGui.button("Collapse##perceptionVisualizers");
```

Reply:
Changed